package.json

```diff
@@ -10,6 +10,10 @@
     "type": "git",
     "url": "git+https://github.com/projectblacklight/blacklight.git"
   },
+  "files": [
+    "app/assets",
```

Comment:
Do we need this?

Reply:
I can say that _we_ at PSU would want to keep it as that is where project blacklight stores it's scss partials that we call from this npm module (see https://github.com/psu-libraries/psulib_blacklight/blob/master/app/javascript/psulib_blacklight/styles/index.scss). Could be done another way by a downstream project I'm sure, but this made sense enough to me.